tests/algorithms/test_density_baselines.py

```diff
@@ -168,3 +170,54 @@ def test_density_trainer_raises(
 
     with pytest.raises(TypeError, match="Unsupported demonstration type"):
         density_trainer.set_demonstrations("foo")  # type: ignore[arg-type]
+
+
+# TODO(GH#794): Remove after https://github.com/DLR-RM/stable-baselines3/pull/1676
+# merged and released.
+class FloatReward(gym.RewardWrapper):
+    """Typecasts reward to a float."""
+
+    def reward(self, reward):
+        return float(reward)
+
+
+def test_dict_space():
+    def make_env():
```

Comment:
Code duplication between this and `test_bc.py`. One potential resolution is to make this a pytest test fixture defined in e.g. `tests/algorithms/conftest.py`